docs/tasks/step-3.md

```diff
@@ -0,0 +1,31 @@
+# Step 3: List all the books in a table
+
+## a) Add table component
+
+On the `/all-books` page list all of the books using a [vuetify data table](https://vuetifyjs.com/en/components/data-tables/basics/).
+
+## c) Add at least one of the following features to the table
+
+### Search
```

Comment:
should these links go to different places e.g.:

https://vuetifyjs.com/en/components/data-tables/data-and-display/#search
https://vuetifyjs.com/en/components/data-tables/data-and-display/#pagination-examples
https://vuetifyjs.com/en/components/data-tables/data-and-display/#filtering-examples

Reply:
Yes they were supposed to, oops, nice spot